Building An SIR Model.rst

```diff
@@ -0,0 +1,29 @@
+
+
+
+Introduction
+------------
+
+Bharatsim is an open-source large-scale agent-based simulation framework that allows you to build and simulate sophisticated agent-based models. Agents form the main component of such a simulation. They are entities that are defined with a set of parameters and obey certain specified conditions as the simulation runs.
+
+In the purview of the SIR model, the agents are people, which go about their daily lives and follow regular schedules. Here, the parameters defining an agent are a unique agent id, their age, their infection status, the time for which they remain infected and so on. The framework allows you to introduce new parameters as and when required.
+
+The rules governing the behaviour of all agents include the particular schedules which they follow and the functions which allow the agents to transit from one compartment to another as described before.
+
+Model Components
+----------------
+
+The model consists of different objects and classes and can be separated into individual ``.scala`` files.
+

```

Comment:
I think you should start of by describing what a Node is, then divide them into the different types (Network, Agent, etc.)

Reply:
I thought the Node would be defined in the 'Framework Basics' section and we could just refer to that.

Reply:
Good point. Maybe describe Agents and Networks.